src/applications/vaos/new-appointment/index.jsx

```diff
@@ -110,16 +113,16 @@ export function NewAppointment() {
           path={`${match.url}/how-to-schedule`}
           component={ScheduleCernerPage}
         />
-        {!hasResidentialAddress && (
+        {(featureCCIteration || hasResidentialAddress) && (
           <Route
             path={`${match.url}/community-care-preferences`}
-            component={CommunityCarePreferencesPage}
+            component={CommunityCareProviderSelectionPage}
           />
         )}
-        {hasResidentialAddress && (
+        {!hasResidentialAddress && (
```

Comment:
I think this would be `!featureCCIteration && !hasResidentialAddress`, that way it's the reverse of the condition for the route above.

Reply:
Done.